static/js/components/VideoPlayer.js

```diff
@@ -78,6 +82,17 @@ export default class VideoPlayer extends React.Component {
           }, true);
         }
       });
+      // Add listeners to each track
+      const player = this.player;
+      for (let track of tracks.tracks_) {
+        track.addEventListener("modechange", function() {
+          sendGAEvent(
```

Comment:
It looks like this line is uncovered, can you write a test for it?

Reply:
This part of the code is all stubbed out - would need an actual real videojs player object being available.